### PR TITLE
remove gpt4 from free custom assistants

### PR DESCRIPTION
**Pull request by fontanierh**

### CODE REVIEW

**Review by philipperolet — DISMISSED** [review]
:+1:

File: front/migrations/20231114_free_users_gpt_4_custom_assistants.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    `
+  );
+
+  const generationConfigIds = generationConfigIdsRes[0].map((g: any) => g.id);
+
+  console.log(`Found ${generationConfigIds.length} agents to update`);
+
+  if (LIVE) {
+    await AgentGenerationConfiguration.update(
+      { modelId: "gpt-3.5-turbo" },
```
Comment (spolu) [inline front/migrations/20231114_free_users_gpt_4_custom_assistants.ts:29]: You want to handle OpenAI and Anthropic separately?

File: front/migrations/20231114_free_users_gpt_4_custom_assistants.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      LEFT JOIN
+        subscriptions ON ws.id = subscriptions."workspaceId"
+      WHERE
+        subscriptions.id IS NULL)
+    `
+  );
+
+  const generationConfigIds = generationConfigIdsRes[0].map((g: any) => g.id);
+
+  console.log(`Found ${generationConfigIds.length} agents to update`);
```
Comment (spolu) [inline front/migrations/20231114_free_users_gpt_4_custom_assistants.ts:25]: Let's pull the workspace and print the plan here to convince ourselves that everything is good

Comment (spolu) [reply front/migrations/20231114_free_users_gpt_4_custom_assistants.ts:25]: (or the workspace subscription)

**Review by lasryaric — DISMISSED** [review]
LGTM

**Pull request merged**